html5lib/tests/test_meta.py

```diff
@@ -1,7 +1,10 @@
 from __future__ import absolute_import, division, unicode_literals
 
 import six
-from mock import Mock
+try:
+    from unittest.mock import Mock
+except:
```

Comment:
```suggestion
except ImportError:
```
See PEP8 or
* https://docs.astral.sh/ruff/rules/blind-except
* https://realpython.com/the-most-diabolical-python-antipattern